packages/core/src/layout/ItemCard/ItemCard.stories.tsx

```diff
@@ -13,86 +13,95 @@
-        label="Button"
-        subtitle="Pretitle"
-        onClick={() => {}}
-      />
-    </Grid>
-  </Grid>
-);
+const text =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
 
-export const Tags = () => (
-  <Grid container spacing={4}>
-    <Grid item xs={12} md={3}>
-      <ItemCard
-        title="Item Card"
-        description="This is the description of an Item Card with Tags"
-        tags={['one tag', 'one tag']}
-        label="Button"
-      />
-    </Grid>
-    <Grid item xs={12} md={3}>
-      <ItemCard
-        title="Item Card"
-        description="This is the description of an Item Card with Tags"
-        tags={['one tag', 'two tag']}
-        label="Button"
-      />
-    </Grid>
-    <Grid item xs={12} md={3}>
-      <ItemCard
-        title="Item Card"
-        description="This is the description of an Item Card without Tags"
-        label="Button"
-      />
-    </Grid>
-  </Grid>
-);
+const useStyles = makeStyles({
+  grid: {
+    gridTemplateColumns: 'repeat(auto-fill, 12em)',
+  },
+  header: {
+    color: 'black',
+    backgroundImage: 'linear-gradient(to bottom right, red, yellow)',
+  },
+});
 
-export const Link = () => (
-  <MemoryRouter>
-    <Grid container spacing={4}>
-      <Grid item xs={12} md={3}>
-        <ItemCard
-          title="Backstage"
-          description="This is the description of an Item Card with link"
-          label="Read More"
-          href="https://backstage.io"
-        />
-      </Grid>
-      <Grid item xs={12} md={3}>
-        <ItemCard
-          title="Backstage @ GitHub"
-          description="This is the description of an Item Card with link"
-          label="Read More"
-          href="https://github.com/backstage/backstage"
-        />
-      </Grid>
-    </Grid>
-  </MemoryRouter>
+export const Default = () => (
+  <>
+    <Typography paragraph>
+      The most basic setup is to place a bunch of cards into a large grid,
+      leaving styling to the defaults. Try to resize the window to see how they
+      rearrange themselves to fit the viewport.
+    </Typography>
+    <ItemCardGrid>
+      {[...Array(10).keys()].map(index => (
+        <Card key={index}>
+          <CardMedia>
+            <ItemCardHeader title={`Card #${index}`} subtitle="Subtitle" />
+          </CardMedia>
+          <CardContent>
+            {text
+              .split(' ')
+              .slice(0, 5 + Math.floor(Math.random() * 30))
+              .join(' ')}
+          </CardContent>
+          <CardActions>
+            <Button color="primary">Go There!</Button>
```

Comment:
thought: For newbies who are copying code, I think it'd be valuable to add the `to` prop with something like https://backstage.io/docs or something, just because the obvious question may be how do I add it to actually make this action button _do something_?  easy for moderate to advanced folks, harder for new ones?  (and the other examples below?)

Reply:
that's very customer focused of you - great idea, adding!